scripts/zones/VeLugannon_Palace/mobs/Brigandish_Blade.lua

```diff
@@ -14,4 +15,5 @@ function onAdditionalEffect(mob, target, damage)
 end
 
 function onMobDeath(mob, player, isKiller)
+    GetNPCByID(ID.npc.QM3):setLocalVar("PillarCharged", 1)
```

Comment:
I think this needs to be followed up by a timer. 

Though, I'm not finding any research, there is a limited window on how long the pillar remains "charged" for? If I'm wrong, no worries and ignore me.

Reply:
Conflicting accounts on this, but I thought this had been debunked.

Reply:
Well, if so, no worries then.